simplivity/resources/external_stores.py

```diff
@@ -0,0 +1,87 @@
+
+URL = '/external_stores'
+DATA_FIELD = 'external_stores'
+
+
+class ExternalStores(ResourceBase):
+    """Implements features available for SimpliVity External store resources."""
+
+    def __init__(self, connection):
+        super(ExternalStores, self).__init__(connection)
+
+    def get_all(self, pagination=False, page_size=0, limit=500, offset=0,
+                sort=None, order='descending', filters=None, fields=None,
+                case_sensitive=True):
+        """
+        Get all external stores
+        Args:
+            pagination: True if need pagination
+            page_size: Size of the page (Required when pagination is on)
+            limit: A positive integer that represents the maximum number of results to return
+            offset: A positive integer that directs the service to start returning
+              the <offset value> instance, up to the limit.
+            sort: The name of the field where the sort occurs
+            order: The sort order preference. Valid values: ascending or descending.
+            filters: Dictionary with filter values. Example: {'name': 'name'}
+                name: The name of the external_stores to return.
+                    Accepts: Single value, comma-separated list, pattern using one or more asterisk characters as a wildcard.
+                omnistack_cluster_id: The name of the omnistack_cluster that is associated with the instances to return
+                cluster_group_id:The unique identifiers (UIDs) of the cluster_groups associated with the external stores to return
+                    Accepts: Single value, comma-separated list
+                management_ip: The IP address of the external store
+                    Accepts: Single value, comma-separated list, pattern using one or more asterisk characters as a wildcard
+                type: The type of external store
+                    Default: StoreOnceOnPrem
+
+        Returns:
+            list: list of resources
+        """
+        return self._client.get_all(URL,
+                                    members_field=DATA_FIELD,
+                                    pagination=pagination,
+                                    page_size=page_size,
+                                    limit=limit,
+                                    offset=offset,
+                                    sort=sort,
+                                    order=order,
+                                    filters=filters,
+                                    fields=fields,
+                                    case_sensitive=case_sensitive)
+
+    def get_by_data(self, data):
+        """Gets ExternalStore object from data.
+
+        Args:
+            data: ExternalStore data
+
+        Returns:
+            object: ExternalStore object.
+        """
+
+        return ExternalStore(self._connection, self._client, data)
+
+
+class ExternalStore(object):
+    """Implements features available for a single External store resources."""
+
```

Comment:
Are we missing OBJECT_TYPE class variable here?

Reply:
Hi @sijeesh ,
Whenever we will implement the refresh method in the class we can add the  OBJECT_TYPE variable.